### ISSUE TITLE
Timestamp is incorrect on "Recent Comments" view

### ISSUE THREAD

**Issue by jaclynperrone** [hammerlab/cycledash#727]
When I create a new comment, the timeline on the Project page says that it was created "4 hours" ago

![time-bug](https://cloud.githubusercontent.com/assets/5685044/8165589/a273d1a0-135f-11e5-9a10-18883c475975.png)

**Comment by danvk** [comment]
@tavinathanson noooo! time zones strike again.

**Comment by tavinathanson** [comment]
Ugh! :-1: 

If this is a regression, then that's on me for not introducing a test that would've caught it. I'll look into this!

**Comment by tavinathanson** [comment]
@jaclynperrone I can't reproduce this locally or on `http://hammerlab-dev3.hpc.mssm.edu`; are you seeing this locally and/or elsewhere?

**Comment by jaclynperrone** [comment]
Yup! I'm seeing it locally. I also noticed the bug during a demo on @danvk's local machine. So..maybe there is something to that

**Comment by ihodes** [comment]
IIRC Tavi had to set the database time zone to UTC?

**Comment by tavinathanson** [comment]
@ihodes Yeah, I'm thinking it's a Postgres settings thing, need to retrace my steps to remember exactly what I did.

**Comment by tavinathanson** [comment]
@jaclynperrone Are you still hitting this issue?

**Comment by jaclynperrone** [comment]
It looks fine on http://cycledash.demeter.hpc.mssm.edu/, but I am having the issue locally. I probably just need to configure something in my db?

**Comment by tavinathanson** [comment]
I'm trying to figure out what you would need to configure, as we're discussing in #819.

Are you seeing the issue on the `cycledash` DB or the `cycledash-test` DB or both? What's your OS X timezone set to? And what's your `timezone` set to in `/usr/local/var/postgres/postgresql.conf`?

**Comment by tavinathanson** [comment]
Also: if you go into your `cycledash` and `cycledash-test` DBs using `psql` or another client, do you see the following:

```
 created       | timestamp with time zone | not null default now()
 last_modified | timestamp with time zone | not null default now()
```

(Or do you get a `timestamp` or `timestamp without timezone` or something?)

**Comment by jaclynperrone** [comment]
I get the error in both db's,
my OSX is set to EDT,
and `timezone` in /usr/local/var/postgres/postgresql.conf is US/Eastern

And when I run `\d user_comments` I also get:
```
created       | timestamp with time zone | not null default now()
last_modified | timestamp with time zone | not null default now()
```

**Comment by tavinathanson** [comment]
Thanks for checking all those things! Just for kicks, can you try changing your `timezone` to `UTC` by editing the `postgresql.conf`, restart Postgres, and see if you still have the issue?

**Comment by jaclynperrone** [comment]
I changed `timezone` to `UTC` in `postgresql.conf` and my timestamps work now! :confetti_ball:

**Comment by armish** [comment]
Resolved by #819.
